Mathlib/CategoryTheory/Category/ULift.lean

```diff
@@ -182,6 +193,12 @@ def AsSmall.equiv : C ≌ AsSmall C where
   unitIso := NatIso.ofComponents fun _ => eqToIso rfl
   counitIso := NatIso.ofComponents fun _ => eqToIso <| ULift.ext _ _ rfl
 
+/-- Assigning to each category `C` the small category `AsSmall C` induces a functor `Cat ⥤ Cat`. -/
+@[simps]
+def asSmall : Cat.{v₁, u₁} ⥤ Cat.{max w₁ v₁ u₁, max w₁ v₁ u₁} where
```

Comment:
In order to minimize imports (cf. large-import tag), could you move this definition to a new file `CategoryTheory.Category.Cat.AsSmall`. I believe the definition should have a `Cat` prefix, like `Cat.asSmallFunctor`.